install_config/configuring_vsphere.adoc

```diff
@@ -214,6 +230,7 @@ kubernetesMasterConfig:
     cloud-config:
     - "/etc/origin/cloudprovider/vsphere.conf"
 ----
+<1> Don't remove your current `apiServerArguments` params just add the `cloud-provider` and `cloud-config` after all of them.
```

Comment:
s/Don't remove your current `apiServerArguments` params.../You do not need to remove your `apiServerArguments` parameters...